src/fluid/prim2con.hpp

```diff
@@ -29,7 +29,7 @@ void signal_speeds(const Real &rho, const Real &u, const Real &p, const Real &bs
   const Real vasq = bsq / (rho_rel + bsq);
   Real cssq = gam1 * p / rho_rel;
   PARTHENON_DEBUG_REQUIRE(rho > robust::SMALL(), "rho is unacceptably small!");
-  PARTHENON_DEBUG_REQUIRE(u > robust::SMALL(), "u is unacceptably small!");
+  //PARTHENON_DEBUG_REQUIRE(u > robust::SMALL(), "u is unacceptably small!");
```

Comment:
Can you try uncommenting this out and running in debug mode? We may not need to comment it out anymore